src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -337,21 +349,6 @@ private void showTrackingNotification() {
         telemetryPreferences.setAskToCollectTelemetry(false);
     }
 
-    /**
-     * The MacAdapter calls this method when a "BIB" file has been double-clicked from the Finder.
```

Comment:
Don't we have the MacAdapter any more? Oh, I did not find it in the current source. Maybe gone? I only found a recent comment at https://github.com/JabRef/jabref/pull/10143#discussion_r1292287331.

Reply:
There's no mac adapter any more it was an old swing thing I think

Reply:
A Solution for this would probably be https://github.com/0x4a616e/NSMenuFX .
I think that the methods removed are of no use for the new approach to fix this with modern mac and javafx, so I removed them. As I have no mac to test this on, I would leave this for someone else with a mac to do.